### ISSUE TITLE
Certain record names will cause the record not to be written to storage.

### ISSUE THREAD

**Issue by aidanok** [deepstreamIO/deepstream.io-storage-rethinkdb#13]
Record names that that map to a table name which contains invalid characters cause writes to storage to silently fail.

To reproduce: 

var rec = ds.record.getRecord('foo-bar/32');
rec.set({x: 1});

At this point the record behaves normally as its in the cache, but it hasn't been written to storage, and on restart or cache eviction the record will be lost.

RethinkDB only allows A-z 0-9 and '_' in the table name which is what is causing this. 
 
Note there is no error message when this happens. ( logLevel: DEBUG )

**Comment by Iiridayn** [comment]
Are you getting "undefined" messages in your output log? Those might be the error messages, masked.

**Comment by aidanok** [comment]
Nope, no 'undefined' messages, just normal logging output at debug level, not indicating any problem.

**Comment by yasserf** [comment]
This is a good find, we'll put it into the next release due end of next week
